src/frontend/src/table/insert.rs

```diff
@@ -120,23 +70,136 @@ pub(crate) fn to_grpc_insert_request(
     })
 }
 
+fn vector_to_grpc_column(
+    table_meta: &TableMeta,
+    column_name: &str,
+    vector: VectorRef,
+) -> Result<Column> {
+    // Safety: timestamp column has to be existed in a table.
+    let semantic_type = if column_name == table_meta.schema.timestamp_column().unwrap().name {
```

Comment:
We have external tables and maybe other table engines in future. So the `unwrap` is a trap in the future. I think we can add an `ensure!` to return an error.